fbchat/graphql.py

```diff
@@ -205,6 +207,7 @@ def graphql_to_group(group):
     if group.get('image') is None:
         group['image'] = {}
     c_info = get_customization_info(group)
+    last_message_timestamp = group['last_message']['nodes'][0]['timestamp_precise']
```

Comment:
This line fails when calling `searchForGroups`, with a `KeyError: u'last_message'`. I've changed it to
```
if 'last_message' in group:
    last_message_timestamp = group['last_message']['nodes'][0]['timestamp_precise']
```